pydicer/convert/rtstruct.py

```diff
@@ -0,0 +1,99 @@
+from pathlib import Path
+import pydicom
+import SimpleITK as sitk
+from matplotlib import cm
+
+from platipy.dicom.io.rtstruct_to_nifti import transform_point_set_from_dicom_struct
+from platipy.imaging.utils.io import write_nrrd_structure_set
+
+
+def convert_rtstruct(
+    dcm_img_list,
+    dcm_rt_file,
+    prefix="Struct_",
+    output_dir=".",
+    output_img=None,
+    spacing=None,
+):
+    """Convert a DICOM RTSTRUCT to NIFTI masks.
+
+    The masks are stored as NIFTI files in the output directory
+
+    Args:
+        dcm_img_list (str|pathlib.Path): Path to the reference DICOM image series
+        dcm_rt_file (str|pathlib.Path): Path to the DICOM RTSTRUCT file
+        prefix (str, optional): The prefix to give the output files. Defaults to "Struct_".
+        output_dir (str|pathlib.Path, optional): Path to the output directory. Defaults to ".".
+        output_img (str|pathlib.Path, optional): If set, write the reference image to this file as
+                                                 in NIFTI format. Defaults to None.
+        spacing (list, optional): Values of image spacing to override. Defaults to None.
+    """
+
+    # logger.debug("Converting RTStruct: {0}".format(dcm_rt_file))
```

Comment:
I guess the logger stuff here can be added back once #41 is implemented

Reply:
Yep once this merged through I'll synchronize back into #41 and fix it up in there.